src/main/java/teammates/ui/controller/InstructorEditStudentFeedbackSaveAction.java

```diff
@@ -69,14 +70,12 @@ protected void checkAdditionalConstraints() {
             boolean isRecipientVisibleToInstructors = questionAttributes.showRecipientNameTo.contains(FeedbackParticipantType.INSTRUCTORS);
             boolean isResponseVisibleToInstructors = questionAttributes.showResponsesTo.contains(FeedbackParticipantType.INSTRUCTORS);
             
-            
-            if (!isGiverVisibleToInstructors || !isRecipientVisibleToInstructors || !isResponseVisibleToInstructors) {
+            if (!isResponseVisibleToInstructors || !isGiverVisibleToInstructors || !isRecipientVisibleToInstructors) {
                 isError = true;
                 throw new UnauthorizedAccessException(
                         "Feedback session [" + feedbackSessionName + 
                         "] question [" + questionAttributes.getId() + "] is not accessible to instructor ["+ instructor.email + "]");
             }
-            
         }
     }
     
```

Comment:
appendRespondant() and removeRespondant() has little coverage but after looking through, still not very sure how we can test those especially removeRespondant?